03_how_ln_works.asciidoc

```diff
@@ -526,7 +526,7 @@ The user might only realize that probing is taking place if the payment does not
 
 [NOTE]
 ====
-On the Internet we use the internet protocol and an IP forwarding algorithm to forward internet packages from the sender to the destination. While these protocols have the nice property of allowing internet hosts to collaboratively find a path for information flow through the internet, we cannot reuse and adopt this protocol for forwarding payments on the Lightning Network. Unlike the internet, Lightning payments have to be _atomic_ and channel balances have to remain _private_. Furthermore, the channel capacity in Lightning changes frequently, unlike the Internet where connection capacity is relatively static. These constraints require novel strategies.
```

Comment:
I think in this section, "packages" is a mistranslation and should be "packets" as in network packets.  I haven't made these changes in case I'm wrong.

Reply:
open an issue #758 so that I can merge!